onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -728,8 +615,9 @@ Status QnnBackendManager::SetupBackend(const logging::Logger& logger, bool load_
     ORT_RETURN_IF_ERROR(LoadQnnSystemLib());
   }
 
+  sdk_build_version_ = GetBackendBuildId();
   LOGS(logger, VERBOSE) << "Backend build version: "
-                        << GetBackendBuildId();
+                        << sdk_build_version_;
```

Comment:
what does this string look like?  is it the qnn version string (in the case of qnn ep) ?

Reply:
v2.14.1.230828143325_61701

---
In reply to: [1348228328](https://github.com/microsoft/onnxruntime/pull/17757#discussion_r1348228328) [](ancestors = 1348228328)

Reply:
thanks